warrior/Framework/ClassUtils/rest_utils_class.py

```diff
@@ -217,22 +217,31 @@ def cmp_response(self, response, expected_api_response,
                                   " provided file path", "error")
                             return False
                     status, sorted_file1, sorted_file2, output_file = \
-                    Utils.xml_Utils.compare_xml(extracted_response,
-                     expected_api_response,
-                                output_file, sorted_json=False)
+                    Utils.xml_Utils.compare_xml(extracted_response, expected_api_response,
+                                                output_file, sorted_json=False)
 
                 elif 'json' in response.headers['Content-Type']:
                     try:
-                        expected_api_response = JSON.load(open(
-                                            expected_api_response, 'r'))
+                        expected_api_response = JSON.load(open(expected_api_response, 'r'))
```

Comment:
If we are going to add the ```ENV``` support, we are better off adding support for ```REPO``` substitutions as well. Please look into this [line](https://github.com/warriorframework/warriorframework_py3/blob/master/warrior/Framework/Utils/data_Utils.py#L420) for the substitution utilities.
We also support changing the start and end patterns - so instead of the standard pattern ```${ }```, we can use anything unique. Please make sure this is supported, too.

Reply:
Hi @sanikakulkarni ,

The issue is raised by the end user and we have given the fix for the user .
I will raise a new ticket for the suggestions what you have given and we will close that ticket in next release .
Can you please let me know , your opinion on this.

Reply:
Hi @sanikakulkarni 

we will use war-630 to address the issue.

Reply:
Hi @apathapa-warrior,

WAR-630 is a unrelated bug in the same space. I would recommend creating a new ticket to add in changes requested in my comment if this ticket is high priority and needs to go in asap.

Reply:
Hi @sanikakulkarni 

I have raised the new ticket war-2134.